### DISCUSSION TITLE
能不能构建出一个直接可以运行的二进制文件哇

### DISCUSSION THREAD

**Discussion by citizenll** [ejoy/ant#32] [General]
想体验一下，环境太麻烦了。梯子网速不给力 子模块根本下不来

**Comment by chucklu** [comment]
try `git submodule update --init --depth 1`

**Comment by glfeng318** [comment]
讨论请到 [Discussions](https://github.com/ejoy/ant/discussions)

**Comment by actboy168** [comment]
https://github.com/actboy168/ant-prebuilt/releases/tag/nightly
